mappings/net/minecraft/structure/pool/StructurePoolBasedGenerator.mapping

```diff
@@ -1,5 +1,14 @@
 CLASS net/minecraft/class_3778 net/minecraft/structure/pool/StructurePoolBasedGenerator
 	FIELD field_16665 LOGGER Lorg/apache/logging/log4j/Logger;
+	METHOD method_30419 generate (Lnet/minecraft/class_5455;Lnet/minecraft/class_3812;Lnet/minecraft/class_3778$class_3779;Lnet/minecraft/class_2794;Lnet/minecraft/class_3485;Lnet/minecraft/class_2338;Lnet/minecraft/class_6130;Ljava/util/Random;ZZLnet/minecraft/class_5539;)V
```

Comment:
Mind update `StructurePoolBasedFeatureConfig`'s javadoc's `method_30419` to `generate` instead? https://github.com/FabricMC/yarn/pull/2559/checks#step:6:3543